docs/docusaurus.config.js

```diff
@@ -121,20 +121,24 @@ const config = {
                 html: '<i class="fa-fw fa-brands fa-github"></i> Github',
               },
               {
-                href: "https://reddit.com/r/cosmosnetwork",
-                html: '<i class="fa-fw fa-brands fa-reddit"></i> Reddit',
-              },
-              {
-                href: "https://www.youtube.com/c/CosmosProject",
-                html: '<i class="fa-fw fa-brands fa-youtube"></i> YouTube',
+                href: "https://twitter.com/cosmoshub",
+                html: '<i class="fa-fw fa-brands fa-twitter"></i> Twitter',
               },
               {
-                href: "https://discord.gg/cosmosnetwork",
+                href: "https://discord.gg/interchain",
                 html: '<i class="fa-fw fa-brands fa-discord"></i> Discord',
               },
               {
                 href: "https://forum.cosmos.network/",
-                html: '<i class="fa-fw fa-regular fa-comments"></i> Cosmos Forums',
+                html: '<i class="fa-fw fa-regular fa-comments"></i> Forum',
+              },
+              {
+                href: "https://reddit.com/r/cosmosnetwork",
+                html: '<i class="fa-fw fa-brands fa-reddit"></i> Reddit',
+              },
+              {
+                href: "https://www.youtube.com/c/CosmosProject",
```

Comment:
Seems last time a video was uploaded there was 2 years ago. Do you want to have this here?

Reply:
That's a discussion for a different PR.